sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/ColumnType.scala

```diff
@@ -705,14 +705,11 @@ private[columnar] case class MAP(dataType: MapType)
   override def clone(v: UnsafeMapData): UnsafeMapData = v.copy()
 }
 
-private[columnar] object CALENDAR_INTERVAL extends ColumnType[CalendarInterval]
-  with DirectCopyColumnType[CalendarInterval] {
+private[columnar] object CALENDAR_INTERVAL extends ColumnType[CalendarInterval] {
 
   override def dataType: DataType = CalendarIntervalType
 
-  override def defaultSize: Int = 16
-
-  override def actualSize(row: InternalRow, ordinal: Int): Int = 20
+  override val defaultSize: Int = 16
```

Comment:
is it better to use `def`? to follow `def dataType`.

Reply:
ok